android/java/res/layout/activity_asset_detail.xml

```diff
@@ -58,15 +58,15 @@
             android:textSize="14sp"
             android:layout_marginStart="16dp"
             android:layout_marginEnd="16dp"
-            android:text="Ethereum Price (ETH)"
+            android:text="@string/eth_price"
             android:textColor="#84889C"/>
 
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginStart="16dp"
             android:layout_marginEnd="16dp"
-            android:text="$1,812.31"
```

Comment:
Strings that don't require translation should be hard coded it in java code and `setText` from there.

Reply:
This is the same thing mentioned above. @deeppandya do you think it is better to ignore these?

Reply:
it should be fine. please leave as is. i will update it with real data.

Reply:
Ok then I will reverse the changes to these strings.

Reply:
They have been reverted.